mk/config.mk

```diff
@@ -337,6 +337,14 @@ CFG_CORE_SANITIZE_KADDRESS ?= n
 # Add stack guards before/after stacks and periodically check them
 CFG_WITH_STACK_CANARIES ?= y
 
+# Use compiler instrumentation to troubleshoot stack overflows: checks stack
+# limits before returning from most C functions and panics immediately if SP is
```

Comment:
Should be "when entering" rather than "before returning from" (I changed the code after writing the comment).